### PR TITLE
[fix] Typo in redirect, Add sponsors

### PR DESCRIPTION
**Pull request by hqasmei**

### CODE REVIEW

**Comment by danstepanov** [comment]
@hqasmei can you please rebase this branch with `beta` and resolve the above conflicts?

- on branch `beta`
`git pull origin beta`
`git checkout fix/links`
`git rebase -i beta`
- in vim, type `:wq`
- resolve conflicts in your editor (VSCode, etc)
- either click "Continue" in VSCode or run `git rebase --continue` in your terminal
- `git push -f origin fix/links`

image

**Comment by hqasmei** [comment]
@danstepanov thanks just did it

File: www/src/components/landing/sponsors/sponsorsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Sponsor } from "./sponsor";
+
+export default function SponsorsSection() {
+  return (
+    <section className="relative mt-12 sm:mt-40 z-[1] w-[90%] lg:w-[70%] sm:w-auto pb-24">
+      <h1 className="text-center text-4xl font-bold tracking-tight text-white md:text-6xl lg:text-[4rem] xl:text-[4rem] pb-6 pt-24 2xl:pt-16">
+        Sponsors
+      </h1>
+      <div className="flex flex-col items-center gap-6">
+        {/* honeycomb grid for the future */}
```
Comment (danstepanov) [inline www/src/components/landing/sponsors/sponsorsSection.tsx:11]: @hqasmei can you remove this unused code

**Review by danstepanov — CHANGES_REQUESTED** [review]
1 nit

**Comment by danstepanov** [comment]
@hqasmei Also, can you reduce the white space here on mobile?
Screenshot 2023-11-23 at 4 05 08 PM

**Review by danstepanov — APPROVED** [review]
LGTM

**Pull request merged**